detect_interval: 3


final_goal: [0.3,-0.4,0,0,1,0]

# find toycar
find_toycar:
  patrol_route: [[0,0,0,0,0,0,1],[1.85,0,0,0,0,0.38,0.92],[1.6,-0.3,0,0,1,0],[0.5,-0.2,0,0,0,-0.38,0.92]]

docking_toycar:
  left_port: [[545,390]]
  right_port: [[763,390]]
  enter_port: 390
# detect model param
detect:
  conf_thres: 0.3
  model_half: True
  model_path: '/home/gloria/catkin_ws/src/push_toycar/model/best.pt'
  device: 'gpu' # 'gpu' or 'cpu'

# camera param
camera:
  image_shape: [1280,720]
  camera_fps: 30
  # camera_dev  and camera_param_root
  far_camera:
    dev: 4
    path: 'camera1'
  near_camera:
    dev: 2
    path: 'camera2'
